src/plugins/dataflow/components/dataflow-tile.tsx

```diff
@@ -206,13 +207,14 @@ export default class DataflowToolComponent extends BaseComponent<IProps, IDatafl
     // Time   |  Node 1 | Node 2 | Node 3 etc
     //    0   |   val    | val    |  val
     addAttributeToDataSet(tileContent.dataSet, { name: "Time (sec)" }); //time quantized to nearest sampling rate
+    let insertionOrder = 0;
     tileContent.program.nodes.forEach((n) => { //add attributes based on nodes in tile
```

Comment:
just fyi, when `(n, idx)` is used  `idx` returns an object at that index within the MST map,  so I had to use a counter variable outside of the forEach loop.

Reply:
When using `forEach` on a map, the arguments to the callback are `(value, key)` where `key` is the key used to store/lookup the `value` in the map. This is actually the same as arrays where the `key` happens to also be the array index. FYI, another way to achieve the index from a map is to use `map.values().forEach((value, idx) => {})`.